tests/e2e/e2e.mk

```diff
@@ -37,3 +37,22 @@ test-e2e-aws-simple-1-20: test-e2e-install
 		--test-package-marker=stable-1.20.txt \
 		--parallel 25 \
 		--skip-regex="\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|Dashboard|RuntimeClass|RuntimeHandler"
+
+.PHONY: test-e2e-do-simple-1-20
+test-e2e-do-simple-1-20: test-e2e-install
+	kubetest2 kops \
+		-v 6 \
+		--up --down \
+		--cloud-provider=digitalocean \
+		--cluster-name e2e-test-do.k8s.local \
+		--env S3_ENDPOINT=sfo3.digitaloceanspaces.com \
+		--env JOB_NAME=pull-kops-e2e-kubernetes-do-kubetest2 \
+		--create-args "--networking=cilium --api-loadbalancer-type=public --master-count=3" \
+		--kops-version-marker=https://storage.googleapis.com/kops-ci/bin/latest-ci-updown-green.txt \
```

Comment:
@rifelpet - is there a way I can use an older kops binary here (say kops binary released for v1.18 instead ?)

Reply:
give this a shot:

https://storage.googleapis.com/k8s-staging-kops/kops/releases/markers/release-1.18/latest-ci.txt